src/util/String.cc

```diff
@@ -110,18 +111,62 @@ vector<uint64_t> String::parseToVectUInt64(string const& str, string const& deli
                                              throwOnError, defaultVal);
 }
 
-string String::toHex(char const* ptr, size_t length) {
+string String::toHex(char const* ptr, size_t length, string const& prefix, bool lowerCase) {
```

Comment:
Should we consider using base64 encoding instead of hex?  Potentially better use of bandwidth, plus standard encoders/decoders available on Python side...

Reply:
Adding base64 is a good idea. The "hex" encoding is an important case since we already depend on it in the Qserv integration tests for processing queries involving binary column types.

Reply:
Eventually, when `qhttp` will be ready to parse the multipart file attachments I will extend the ingest API to accept CSV-encoded payloads in the request body. This would be the most efficient way to send data & schema to the REST API. The present JSON-based packaging was just a good (and simple) way to start based on existing functionality and interfaces (`qhttp`).